apps/cli/src/commands/build.ts

```diff
@@ -28,7 +29,8 @@ const CARTESI_LABEL_DATA_SIZE = `${CARTESI_LABEL_PREFIX}.data_size`;
 const CARTESI_DEFAULT_RAM_SIZE = "128Mi";
 
 const CARTESI_LABEL_SDK_VERSION = `${CARTESI_LABEL_PREFIX}.sdk_version`;
-const CARTESI_DEFAULT_SDK_VERSION = "0.6.0";
+const CARTESI_LABEL_SDK_NAME = `${CARTESI_LABEL_PREFIX}.sdk_name`;
+const CARTESI_DEFAULT_SDK_VERSION = "0.7.0";
```

Comment:
Since the PR #26  suggests a patch to cartesi/sdk, this should be:

```suggestion
const CARTESI_DEFAULT_SDK_VERSION = "0.6.2";
```